addons/mrp/report/mrp_report_bom_structure.py

```diff
@@ -591,6 +594,7 @@ def _format_route_info(self, rules, rules_delay, warehouse, product, bom, quanti
                 'route_name': manufacture_rules[0].route_id.display_name,
                 'route_detail': bom.display_name,
                 'lead_time': bom.produce_delay + rules_delay + manufacturing_lead + bom.days_to_prepare_mo,
+                'lead_time_without_dtp': bom.produce_delay + rules_delay + manufacturing_lead,
```

Comment:
Why don't use `manufacturing_delay` key ?

Reply:
My main concern is that there are many dependencies between these values, and `manufacture_delay` is already used in `_get_lead_days` in stock rules, so I preferred to introduce a new key. I hoped that, during a review, someone will help me figure out a better name :P  

However, it seems that `manufacture_delay` might actually be the right field to use here, assuming it represents the actual lead time without days to prepare. I pushed a test commit to replace `lead_time_without_dtp` with `manufacture_delay` and see if the tests are passing. If so, I will fixup the new commit.

Reply:
The problem with the old `lead_time` is that it mixes different manufacture times, and then at some point we subtract the days to prepare from it to get the actual lead time. But the calculations are done a bit differently in other modules, and the resulting `lead_time` is not always actual lead time + days to prepare. If these custom calculations (for subcontracting and purchase) don't break the existing `manufacture_delay` logic, then I think we can use that :)

Reply:
@Whenrow I've updated the code. Do you think modifying `manufacture_delay` could have some negative effects on other calculations? Should I add more tests?

Reply:
To me it's ok to use a value that already exist. Especially a value that "represent" something similar that the key you want to add.